OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -18,7 +18,9 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
-		static Player FindPlayerByClient(this World world, Session.Client c)
+    private const string BattlefieldControlChatName = "Battlefield Control";
```

Comment:
ok, thanks. I close this PR, as I created new one because I deleted my fork in the meantime. not sure, how to better handle it.